testdata/pretty-microerror-1-depth-annotation-stack-trace.golden

```diff
@@ -0,0 +1,3 @@
+Error: Something went wrong: Something bad happened, and we had to crash
+
+--REPLACED--/pretty_test.go:161
```

Comment:
I thought that we may want to skip `Error: ` prefix and print that in the printer concept instead? Sometimes we'll print warnings because of retries etc.